pkg/framework/framework.go

```diff
@@ -26,39 +26,82 @@ type Framework struct {
 	JvmbuildserviceController *jvmbuildservice.SuiteController
 }
 
+type Framework struct {
+	Appstudio     *ControllerHub
+	Hacbs         *ControllerHub
+	AppstudioUser *ControllerHub
+	HacbsUser     *ControllerHub
```

Comment:
`HacbsUser > HacbsUserWs`
`AppstudioUser > AppstudioUserWs`

Better to include Workspace naming here to not get lost the people...

Reply:
updated